src/BaseCurve.sol

```diff
@@ -13,16 +14,28 @@ import {UD60x18, ud60x18} from "@prb/math/UD60x18.sol";
  *      accomodating for the effect of fees, supply burn, airdrops, etc) are handled by the EthMultiVault instead
  *      of the curves themselves.
  */
-abstract contract BaseCurve {
+abstract contract BaseCurve is IBaseCurve {
```

Comment:
3.18 - Missing Inheritance From Interface

We made the BaseCurve inherit from the IBaseCurve interface.